configs/predict.yaml

```diff
@@ -5,6 +5,7 @@ defaults:
   - _self_
   - dataset: conll2003
   - serializer: json
```

Comment:
did we miss to add `hydra` here?

Reply:
I guess predict was little half baked since there was nothing to compare it with from original repo. However, I have updated it now according to train and test configs.